Automattic/wp-calypso
# Adds Offer Reset constants

#### Changes proposed in this Pull Request

* Adds Offer Reset constants and tests them with new plan picker.

#### Testing instructions



* Verify constants and text matches copy document (see ticket below)

Fixes 1169247016322522-as-1187323761633610.
